lib/travis/api/v3/queries/log.rb

```diff
@@ -8,7 +8,7 @@ def find(job)
       raise EntityMissing, 'log not found'.freeze if log.nil?
       #if the log has been archived, go to s3
       if log.archived_at
-        content = fetch.get(prefix).try(:body)
+        content = fetch.get(prefix).try(:body).force_encoding('UTF-8')
```

Comment:
`travis-logs` has coder in Gemfile: https://github.com/travis-ci/travis-logs/blob/master/Gemfile#L8

And cleans strings: https://github.com/travis-ci/travis-logs/blob/master/lib/string_cleaner.rb

I think it's encoding as Base64?
https://github.com/travis-ci/travis-logs/blob/master/lib/travis/logs/app.rb#L106
https://github.com/travis-ci/travis-logs/blob/master/spec/integration/app_spec.rb#L205